config/standalone-config.properties

```diff
@@ -10,38 +10,64 @@
 
 ##region General Segment Store Settings
 
-# Port on which a ZK server is started
+# Port on which a ZK server is started.
+# Default value: 4000
 #singlenode.zkPort=4000
 
 # Port on which segmentstore server is started.
+# Default value: 6000
 #singlenode.segmentstorePort=6000
 
 # Port on which the controller server grpc is started.
+# Default value: 9090
 #singlenode.controllerPort=9090
 
-# Enable REST server.
-#singlenode.enableRestServer=true
+# Whether to enable REST server.
+# Default value: false
+singlenode.enableRestServer=false
 
 # Port on which the controller REST service is started.
+# Default value: 9091
 #singlenode.restServerPort=9091
 
-# Cert file for TLS communication.
+# Location of server's certificate file for TLS communication.
+# Default value: ../config/cert.pem
 #singlenode.certFile=../config/cert.pem
 
-# Key file for TLS communication
+# Location of server's key file for TLS communication.
+# Default value: ../config/key.pem
 #singlenode.keyFile=../config/key.pem
 
 # Password file for default password handler
+# Default value: ../config/passwd
 #singlenode.passwdFile=../config/passwd
 
-#Username for default password handler implementation
+# Username for default password handler implementation.
 #singlenode.userName=admin
 
-#Password associated with the default user
+# Password associated with the default user
 #singlenode.passwd=1111_aaaa
 
-# Enable TLS for segmentstore and controller.
-#singlenode.enableTls=true
+# Whether to enable TLS for segmentstore and controller.
+# Valid values: 'true' or 'false'
+# Default value: false
+singlenode.enableTls=false
 
-# Enable authentication/authorization.
+# Whether to enable authentication/authorization.
+# Valid values: 'true' or 'false'
+# Default value: false
 #singlenode.enableAuth=true
```

Comment:
This parameter may cause confusion, as `true` and `false` appear here as possible defaults. Moreover, in `SingleNodeConfig.java` we still have `public final static Property<Boolean> ENABLE_AUTH = Property.named("enableAuth", false);`. It would be nice to clarify this.

Reply:
@Raul - The thinking was that since the default value for this boolean propery is false, one'd really need to change it only for setting it to true. So, why not just set it to "true" in that case, so that the user only needs to uncomment the line if he wants to change the default behavior. 

That said, if our convention is to set the property values to default values even if the property is commented out, it'd make more sense to follow that convention and change that line to #singlenode.enableAuth=false. 

What do you say?

Reply:
@raul, I've now pushed a change that uses the default value as the value in the commented out property.